Makefile

```diff
@@ -39,6 +39,7 @@ test:
 
 integration:
 	go test -mod=vendor -count=1 -v -cover -race -run TestIntegration ./...
+	go test -mod=vendor -count=1 -v -cover -race -run Example_clusterBackgroundTask ./pkg/routine
```

Comment:
Is there a specific reason why we call it an Example?

Reply:
Nvm, you are using the example output feature of go. I see.

Reply:
Because it's an example how to use it.